providers/base/bin/memory_compare.py

```diff
@@ -118,6 +199,8 @@ def main():
         print("Results:")
         print("\t/proc/meminfo reports:\t{}".format(visible_memory))
         print("\tlshw reports:\t{}".format(installed_memory))
+        print("\tFound FW allocated %d MB for PCI GPUs" %
+                fw_allocated_memory_for_pci)
         print("\nPASS: Meminfo reports %s less than lshw, a "
```

Comment:
The `difference` might be from meminfo, but it might also be from meminfo **and** the memory allocated for the GPU, so this message should be modified to fit the reported info.

(the same applies below when there is an error)